### PR TITLE
Some tweaks

### PR DESCRIPTION
**Pull request by claudioandre**
.

### CODE REVIEW

File: src/common-opencl.c [hunk 1]
```diff
@@ -211,12 +211,11 @@ int get_sequential_id(unsigned int dev_id, unsigned int platform_id)
...
 }
 
-static char *opencl_driver_ver(int sequential_id)
+void opencl_driver_value(int sequential_id, int * major, int * minor)
 {
-	static char ret[64];
 	char dname[MAX_OCLINFO_STRING_LEN];
 	char *p;
-	int major = 0, minor = 0;
+	*major = 0, *minor = 0;
```
Comment (magnumripper) [inline src/common-opencl.c]: What's this? void pointers? int pointers?

Comment (magnumripper) [inline src/common-opencl.c]: Ah no they are arguments, sorry

**Comment by magnumripper** [comment]
> I don't think you should start removing #if FMT_MAIN_VERSION > 11 #endif in individual formats, 

Actually since we have released Jumbo-1, I think we can remove them across the board. But you are right it's better done in a separate commit.

**Comment by frank-dittrich** [comment]
> Actually since we have released Jumbo-1, I think we can remove them across the board. But you are right it's better done in a separate commit.

Then let's create a new issue, and plan to cleanup all the other dependencies FMT_MAIN_VERSION:
```
    593 #if FMT_MAIN_VERSION > 11
     30 #if FMT_MAIN_VERSION > 9
      4 #if FMT_MAIN_VERSION > 10
      1 #if FMT_MAIN_VERSION > 11 && IKE_REPORT_TUNABLE_COSTS
```

This could be an easy part of the GSoC code cleanup task.

**Comment by magnumripper** [comment]
> Anyone can create xsha512 hashes for real, so I can do a real test?

We have passgen.pl, in the run directory. It supports a lot of formats but not all.
```
$ fortune | sed 's/ /\n/g' | ../run/pass_gen.pl xsha512
#!comment: Built with pass_gen.pl using RAW mode, 0 to 128 characters dict file=unknown

  ** Here are the hashes for format  **
u0:5139424313bda9648a17811e171397ace942c2f459e82ea4594cabae9e98310cec52c6e3e2a04be63d97f15685d8ffd044c5d1feda93aebd9157ac1e7573ec931f4b2644:0:0:I:
u1:$LION$745a3775f83446811c0de57664318d8137108a78fb947c15afe4bda279085aeda1578fcb030295f704451ab8416d8e0064c7bc9f7ab9f172b3a017e86c634cff5582e7b6:1:0:get:
u2:574d703849308ce89adc34c4ea6b6bda5da42b8ee7d824f2ace5ffa2c54266c80ddf18a2777dbb6b987c407840427fe7108da5f905a1ac57f8fbe811242b0298b9542dca:2:0:my:
u3:$LION$714f7330806bce90beea9f709e57e675463b4075f4981edf334bc32acdc14dc2be0ea98022076bf37191a47c284f70b77f1bafd607e12b4e2140a29f435310c521b901e2:3:0:exercise:
u4:4942767965cb82fdd2902f7fe545e2ca6801f7fb3bdbe548a6a77818807e26b739933bf62abf6a9e5b87e512ab67d486915e120295faba07e450b0d0c06a688e32dfa42f:4:0:acting:
u5:$LION$4a783337a6a75c82e99cb10a2475402ea1e735551f502d8e58a9019844ceec59353585c04980a44ad5e49a82fe2ff3f20823c7bef8c138f38eeb4b2f13bddb009124f571:5:0:as:
u6:5367364301a63112f374df26f28f1018fb5c58d7eda83803973955215c2c96be2f4dcc76335e92c780d9c34efab1fe1ca72aba1a48441b23e290812c42ef42c098054f6a:6:0:pallbearer:
u7:$LION$52367137c33e6629849751c880b3214ace2e04db6d1f934933e51b7b1d557323c41115c3ec2ed2f0d0e99798f0323f048f5fd1f69bc4adbd71e1cb3ae163086dc410af68:7:0:to:
u8:4f63484ed5e64844a00f5232193c5a4fa72fa51f37040374770ff5999c22dacde304a3d233f0f95021ef2d89db5a4ef5477b48158d288dddbf26eb18b53b1d0c0bb31ad9:8:0:my:
u9:$LION$6c7a39730d812f94148261e193d811b410a574dbee6848d8b120e4eb3cf4c12e1f80aee93bb63c1c226be87e2bfe8bcc520886ea9d4c9f71e27e43a0e0e42e759c84b0d9:9:0:friends:
u10:58753950c6bc8dcdae5cf73049c8e09b3cd93c03448720de34d54569dbf8b16faaf5a192a933664b75d9ec1ea9f4834bbfdc3cf432cd43bf76bc917ffe349c36c6885349:10:0:who:
u11:$LION$54374c6487d4286aa3ffceac6c35941019e73df062a8a69792fa943f9b4de86e5984d159ff3d94e40cd62d55d671887fdaa661b0cf62dcc0e67ee0325db2964ca5adaac3:11:0:exercise.:
u12:674777429c902624529d40d6ed490cc3c0da47979cdea3e8406d33f1c1280dffb239e0a09dcc4b1f023787aa5b8b223f796544e5055911cef764fac554f880b046df0cc7:12:0:		--:
u13:$LION$76575275a8a20fdeb036ad62c2ea14b914b0c754a06b0a089bae349f8b044cd21e91b06060d92d3d10be21b5ff90634b22b87bd9a41be9319be198a901817bb7a3cc8092:13:0:Chauncey:
u14:4b5373358b85a345918852a502988fc0fc2b944b3b14a562b924a18dd1654cbc196f2d2a854499b22c7914a84cb33dcc83918cb3bfd5da87e0ddd1d9de3116c1a2664cd7:14:0:Depew:
```

**Pull request merged**